_categories/treatments-immunity.md

```diff
@@ -31,5 +31,8 @@ questions:
 - will-miracle-mineral-solution-cure-covid-19
 - what-is-fda-doing-to-protect-people-from-fraud
 - i-built-a-diy-ventilator-may-i-sell-it
-title: Treatments, vaccines, and immunity
+redirect_from:
+- /treatments-vaccines-immunity/
+- /basics/
```

Comment:
It looks like there are now two redirects from `/basics/` - this one, and `covid-19-facts`. Should that one be removed?

Reply:
Hmm. Good catch. It was in two places already, I guess, since it was on treatments-vaccines-immunity (which is why I ported it over). I think it's more relevant to facts so it can be removed from treatments-immunity.

Reply:
Just pushed up a fix. Will test and merge into preview.